ui/app/components/clients/dashboard.js

```diff
@@ -24,6 +24,11 @@ export default class Dashboard extends Component {
   @tracked startYear = null;
   @tracked selectedNamespace = null;
 
+  // conditionals for filtering HBS file
+  // TODO update values depending on data returned from API
+  @tracked isDateRange = true; // false when viewing single month (true on init b/c defaults to billing period)
```

Comment:
we hope 🤞 . If it turns out we cannot reliable get the billing period from an endpoint then I assume (🤷‍♀️) this could be false? Basically, what should this value be if we don't have a billing period start date at all, and therefor no data, just an empty state telling them they need to select a start date.

Reply:
Yeah - I wondered about this. None of the child components will show if there is no data / no date. (They will all have empty state elses if data isn't coming in properly). Based on your comment, I think it'd be better to just leave both of these tracked properties with no default and have them set based on what is inputed into the date selector.